README.md

```diff
@@ -148,14 +196,14 @@ Write ruby code that will create a person.
 
 Your answer:
 ```
-Replace this with your answer
+person = Person.create(id: 1, name: "brad", age: 22)
 ```
```

Comment:
the `id` will automatically get generated any time your create a person through AR, you shouldn't try to manually create a person with an ID.